specs/fjord/derivation.md

```diff
@@ -48,3 +61,51 @@ be in the same epoch as the the last pre-Fjord blocks, even if these blocks woul
 have these L1-origin timestamps according to pre-Fjord rules. So the same L1 timestamp would be
 shared within a pre- and post-Fjord mixed epoch. This is considered a feature and is not considered
 a security issue.
+
+## Increasing `MAX_RLP_BYTES_PER_CHANNEL` and `MAX_CHANNEL_BANK_SIZE`
```

Comment:
Nit: Do these have fork-specific names, like `MAX_RLP_BYTES_PER_CHANNEL_FJORD` and `MAX_CHANNEL_BANK_SIZE_FJORD`? Since the old values still apply for pre-fjord derivation, we need them both..

Reply:
Good point. Do we usually follow the pattern in such circumstances to introduce new constant names? We could instead not introduce fork-specific names for constants, but start treating all "constants" as just protocol "parameters" that might be subject to change in future protocol upgrades. So in the main protocol description, we can name something just `max_rlp_bytes_per_channel` and then additionally describe in individual forks what changes. Implementations will then probably have such fork-specific global constants, but these would be implementation details.

Reply:
I'm not adding the `FJORD` name to the specs because it's easier to to explain there is a limit & that the limit is changing.